vaadin-menu-bar-flow/pom.xml

```diff
@@ -27,11 +27,6 @@
             <artifactId>vaadin-button</artifactId>
             <version>2.1.4</version>
         </dependency>
-        <dependency>
-            <groupId>org.webjars.bowergithub.vaadin</groupId>
-            <artifactId>vaadin-context-menu</artifactId>
```

Comment:
Why is this removed? Doesn't it break the "Fail if non-pinned dependencies found"-check in snapshot build?

Reply:
vaadin-context-menu-flow defines the webjars version of vaadin-context-menu. Why would we want to override that version with a perhaps wrong version of the webjar?

Reply:
Because of the requirement to pin all webjar dependencies. I tested locally the "Fail if non-pinned dependencies found"-script and it fails without this dep. So the snapshot build will keep failing as long as we have that check in place.

Reply:
Can't see how it can be seen a unpinned as it is pinned in the dependency... Also where is this magic script located?

Reply:
It's [here](https://bender.vaadin.com/admin/editRunType.html?id=buildType:Flow_ComponentSnapshot_MenuBar&runnerId=RUNNER_287&cameFromUrl=%2Fadmin%2FeditBuildRunners.html%3Fid%3DbuildType%253AFlow_ComponentSnapshot_MenuBar%26init%3D1&cameFromTitle=), the second build step in all component snapshot builds.